bin/resources/GameIndex.yaml

```diff
@@ -42474,6 +42474,10 @@ SLUS-20467:
         // Fix ingame SPS by interchanging vclipw.xyz vf5, vf5w with cfc2.
         patch=1,EE,001d90e0,word,48489000
         patch=1,EE,001d90e4,word,4bc529ff
+        author=Agrippa
```

Comment:
Comment should read

 // Force English - else the game crashes with French or Spanish set
 //  in the BIOS configuration - a game bug, not an emulator bug.